sdk/lib/rtl/amd64/unwind.c

```diff
@@ -650,6 +642,16 @@ RtlVirtualUnwind(
         }
     }
 
+    /* Check for chained info */
+    if (UnwindInfo->Flags & UNW_FLAG_CHAININFO)
+    {
+        /* See https://docs.microsoft.com/en-us/cpp/build/exception-handling-x64?view=msvc-160#chained-unwind-info-structures */
+        FunctionEntry = (PRUNTIME_FUNCTION)&(UnwindInfo->UnwindCode[(UnwindInfo->CountOfCodes + 1) & ~1]);
+        UnwindInfo = RVA(ImageBase, FunctionEntry->UnwindData);
+        i = 0;
+        goto RepeatChainedInfo;
```

Comment:
In theory this can lead to infinite loops?
If one entry points to itself and has this flag.

Reply:
Well, broken binaries lead to broken behaviour. Is that a problem ?